oleggerasimovich/parser.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import os
+from argparse import Namespace
+
+import psycopg2
+from dotenv import load_dotenv
+
+
+def cli() -> Namespace:
+
```

Comment:
Try not to have a new line between the function definition and the function body (unless the function body is pretty big and hard to distinguish from the function definition).